hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -916,6 +944,8 @@ protected abstract Option<Pair<String, CompactionOperation>> getPendingCompactio
    */
   abstract Stream<HoodieFileGroup> fetchAllStoredFileGroups(String partitionPath);
 
+  abstract Stream<Pair<String, List<HoodieFileGroup>>> fetchAllStoredFileGroups(List<String> partitionPath);
```

Comment:
we can probably avoid some of these additional methods if above suggestion is followed.